backend/src/Repository/PlantRepository.php

```diff
@@ -0,0 +1,56 @@
+<?php
+
+namespace App\Repository;
+
+use App\Entity\Plant;
+use Doctrine\Bundle\DoctrineBundle\Repository\ServiceEntityRepository;
+use Doctrine\Persistence\ManagerRegistry;
+
+/**
```

Comment:
I don't understand anything about PHP, however, it looks like you have a lot of comments in your files. I'd delete them before merging.

Reply:
As far as i understand it, the comments are helpful for the IDE and the code quality tools